src/http/client.rs

```diff
@@ -4,12 +4,13 @@ use alloc::collections::BTreeMap;
 use alloc::string::String;
 use alloc::string::ToString;
 
-use ::log::*;
+use log::*;
```

Comment:
Why?

Reply:
Sorry, my auto cargo fmt in vscode must have changed that. Just fixed it!